freelist_hmap.go

```diff
@@ -4,7 +4,11 @@ import "sort"
 
 // hashmapFreeCount returns count of free pages(hashmap version)
 func (f *freelist) hashmapFreeCount() int {
-	// use the forwardMap to get the total count
+	_assertVerify(int(f.freePagesCount) == f.hashmapFreeCountSlow(), "freePagesCount is out of sync with free pages map")
```

Comment:
This code now will be executed always (so no performance benefit). 

It's the reason I proposed wrapping it in a func: 
```
_assertVerify(func() { return int(f.freePagesCount) == f.hashmapFreeCountSlow() } )
```

Reply:
Sorry for that, fix it~